static/css/photon-site/footer.less

```diff
@@ -84,6 +84,7 @@
 
 .Footer-links-header {
     margin: 40px 0 0;
+    font-weight: 700;
```

Comment:
@eviljeff Thats not obsolete because thats for the headers with some link for example the first section header `Addons` is linked to `addons.mozilla.org`, but `Browsers` and `Products` are not linked to anything so need to add `font-weight: 700` explicitely